es-app/src/CollectionSystemManager.h

```diff
@@ -16,6 +16,8 @@ enum CollectionSystemType
 	AUTO_ALL_GAMES,
 	AUTO_LAST_PLAYED,
 	AUTO_FAVORITES,
+	AUTO_HIDDEN,
```

Comment:
I haven't read all the changes yet, but what's the use case for this one? I mean, is there a use case for a collection with all the hidden files? I probably expected the opposite?

Reply:
I meant to supply this to the user as a way to manage the hidden systems, so to select the ones a user wants to be hidden when not in UImode = Full. Same for the Kidsgames. It will be the UI modes that eventually govern the visibility of these items, but I think we need a way to quickly add/remove items from those sets/collections.

Reply:
Ah, got it. I did think you had an idea for these. I think in concept that's a fine approach - curious about how you envision the user selecting the collection to change at each time, are you re-using the approach currently in place where you'd navigate to... a "hidden" collection, and choose to edit that collection?

Reply:
Yes, that is basically it, I am hoping that is a natural enough way to manage these collections.
( I 'm signing off for tonight, early day tomorrow)

Reply:
Sounds good. My only thought, I suppose, is whether it makes sense for the user to have a dedicated collection (i.e. prime real-estate) for items that are meant to be not visible to the user, though I suppose it depends on what you envision the "hidden" items to be :)

Have a good night, we'll keep the conversation going!

Reply:
I agree there is little-to-no use for the collection game-list as such. My idea was that the user would only enable these collections during a 'management-session', and would disable the collection afterwards again. Because these values are stored in the games' metadata, the filtering would still work beyond that session.
We could go as far as to only allow these particular collections in UImode == full, but I'm not sure that's worth the effort.

Reply:
Of course. Glad you're thinking about that. I suppose when we get to try it out we'll see how it works :)

If "hidden" effectively means "does not show in kiosk mode", is "hidden" the best name for it? I initially assumed it was just support for hiding BIOS roms and such (which I imagine it can be), but it's actually broader?

Just a thought, as usual. :)

Reply:
Good point, there is power in names, so lets reconsider how to present this in the UI. What about `ITEMS TO HIDE` as a name for the collection? Still a bit awkward, I think.
If we really think it is a confusing situation, maybe we could introduce an infopopup that explains the use of this collection?